ginga/rv/plugins/LineProfile.py

```diff
@@ -275,45 +303,55 @@ def _slice(self, naxes, mk):
 
         # For axis > 3
         for i in range(2, naxes):
-            slice_obj[i] = int(round(self.image.revnaxis[i-2] + 1))
+            if i != i_sel:
+                slice_obj[i] = 0  # int(round(self.image.revnaxis[i-2] + 1))
```

Comment:
I get `IndexError` for my test image. It has the following extensions but the cube of interest is in EXT 1.
```
Filename: nrca1_47Tuc_subpix_dither1_newpos_uncal.fits
No.    Name         Type      Cards   Dimensions   Format
  0  PRIMARY     PrimaryHDU     133   ()      
  1  SCI         ImageHDU        10   (2048, 2048, 10, 1)   float32   
  2  PIXELDQ     ImageHDU        10   (2048, 2048)   int32 (rescales to uint32)   
  3  GROUPDQ     ImageHDU        10   (2048, 2048, 10, 1)   uint8   
  4  ERR         ImageHDU        10   (2048, 2048, 10, 1)   float32   
  5  ZEROFRAME   ImageHDU         9   (2048, 2048, 1)   float32   
  6  ASDF        ImageHDU         7   (3495,)   uint8   
```